examples/webgl_geometry_teapot.html

```diff
@@ -363,6 +368,14 @@
 
 			}
 
+			updateStats();
```

Comment:
I think i should have been more precise when writing the corresponding issue text.

@shitian-ni Sry if this causes some additional work now.